### ISSUE TITLE
Self-build for macOS (x86/aarch64)

### ISSUE THREAD

**Issue by eric1932** [dyc3/steamguard-cli#170]
Please consider including macOS rebuilt in releases.

What I did
```bash
rustup target add x86_64-apple-darwin
cargo build --release
cargo build --release --target=x86_64-apple-darwin
# create universal binary
cd targets; lipo x86_64-apple-darwin/release/steamguard-cli release/steamguard-cli \
  -create -output ./steamguard-cli
```

prebuilt binary (commit hash 8c4535): [steamguard-cli.zip](https://github.com/dyc3/steamguard-cli/files/9702361/steamguard-cli.zip)
USE AT YOUR OWN RISK

**Comment by dyc3** [comment]
I'm gonna remove your download here because it presents a risk to end users. But yeah, maybe I should include a darwin build

**Comment by eric1932** [comment]
thx for the update

**Comment by tayler-king** [comment]
`aarch64-apple-darwin` would also be useful :)

**Comment by dyc3** [comment]
I took a look at adding this to my release script, and it's not exactly as straightforward as I'd hoped. For whatever reason, `ring` (transient dependency) really doesn't like getting cross compiled for mac on linux.

**Comment by dyc3** [comment]
yeah but that's a lot of work and im stretched pretty thin rn. PRs welcome
